source/sdk/Memory.ooc

```diff
@@ -2,17 +2,8 @@ include string // Memory routines in C are actually in string.h
 version(windows) { include malloc }
 else { include alloca }
 
-// GC_MALLOC zeroes the memory, so in the non-gc version, we prefer to use calloc
-// to the expense of some performance. Don't use malloc instead - existing ooc code
-// is written assuming that gc_malloc zeroes the memory.
-gc_malloc: func (size: SizeT) -> Pointer { gc_calloc(1, size) }
```

Comment:
I also need the ability to track allocations when debugging. I would rather expand the functionality with debug calls each time memory is allocated or freed so that I can find out if an object exists when crashing.